<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QxrdWelcomeWindow</class>
 <widget class="QMainWindow" name="QxrdWelcomeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QXRD Welcome</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>844</width>
         <height>514</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QWidget" name="widget" native="true">
          <layout class="QGridLayout" name="m_GridLayout">
           <item row="2" column="0">
            <widget class="QCommandLinkButton" name="m_OpenExistingExperiment">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Open an existing Experiment...</string>
             </property>
             <property name="icon">
              <iconset resource="qxrdresources.qrc">
               <normaloff>:/images/fileopen.png</normaloff>:/images/fileopen.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QCheckBox" name="m_AutoOpenRecent">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Automatically open most recent experiment on startup</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCommandLinkButton" name="m_NewExperiment">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Start a new Experiment...</string>
             </property>
             <property name="icon">
              <iconset resource="qxrdresources.qrc">
               <normaloff>:/images/list-add.png</normaloff>:/images/list-add.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>233</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>870</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="m_ActionNewExperiment"/>
    <addaction name="separator"/>
    <addaction name="m_ActionOpenExperiment"/>
    <addaction name="separator"/>
    <addaction name="m_ActionEditApplicationPreferences"/>
    <addaction name="separator"/>
    <addaction name="m_ActionExitApplication"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="m_ActionOpenExperiment">
   <property name="text">
    <string>Open Experiment...</string>
   </property>
  </action>
  <action name="m_ActionEditApplicationPreferences">
   <property name="text">
    <string>Edit Application Preferences...</string>
   </property>
  </action>
  <action name="m_ActionExitApplication">
   <property name="text">
    <string>Exit Application</string>
   </property>
  </action>
  <action name="m_ActionNewExperiment">
   <property name="text">
    <string>New Experiment...</string>
   </property>
   <property name="toolTip">
    <string>New Experiment</string>
   </property>
  </action>
  <action name="m_ActionOpenRecentExperiment">
   <property name="text">
    <string>Open Recent Experiment</string>
   </property>
   <property name="toolTip">
    <string>Open Recent Experiment</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="qxrdresources.qrc"/>
 </resources>
 <connections/>
</ui>
